modules/gapi/include/opencv2/gapi/cpu/video.hpp

```diff
@@ -17,9 +21,209 @@ namespace cpu {
 GAPI_EXPORTS GKernelPackage kernels();
 
 } // namespace cpu
+
+struct KalmanParams
```

Comment:
Classes in public headers should be properly documented.

Reply:
Ok. Applied.